src/cc_toolchain/BUILD

```diff
@@ -0,0 +1,257 @@
+# -*- python -*-
+
+# Copyright 2018-2019 Josh Pieper, jjp@pobox.com.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load(":cc_toolchain_config.bzl", "cc_toolchain_config", "cc_toolchain_config_stm32")
+
+# Create environments for each CPU this toolchain supports
+environment(name = "k8")
+
+environment(name = "stm32f0")
+
+environment(name = "stm32f4")
+
+environment(name = "stm32g4")
```

Comment:
Can we drop support for the `stm32g4` since we don't use it? Between the f4 and h7 we have enough examples otherwise of how stuff is done. This goes for the entire PR.

Reply:
Agreed, removed.